lrtable/src/lib/statetable.rs

```diff
@@ -554,7 +554,7 @@ fn resolve_shift_reduce<StorageT: 'static + Hash + PrimInt + Unsigned>(
     pidx: PIdx<StorageT>,
     stidx: StIdx, // State we want to shift to
     shift_reduce: &mut Vec<(TIdx<StorageT>, PIdx<StorageT>, StIdx)>,
-    shift_stidx: StIdx // State in which the conflict occured
+    conflicts_stidx: StIdx // State in which the conflict occured
```

Comment:
Is it `conflicts_idx` plural or just `conflict_idx`?

Reply:
Yes, should probably be singular. But I think it should still be `conflict_stidx` as it is the id of the state the conflict occurs in.

Reply:
`conflict_stidx` works for me.